liquidator/liquidator.js

```diff
@@ -42,6 +42,29 @@ class Liquidator {
         isValid: x => {
           return toBN(x).lt(toBN(toWei("1"))) && toBN(x).gte(toBN("0"));
         }
+      },
+      liquidationDeadline: {
+        // `liquidationDeadline`: Aborts the liquidation if the transaction is mined this amount of time after the
+        // EMP client's last update time. Denominated in seconds, so 300 = 5 minutes.
+        value: 300,
+        isValid: x => {
+          return x >= 0;
+        }
+      },
+      liquidationMinPrice: {
+        // `liquidationMinPrice`: Aborts the liquidation if the amount of collateral in the position per token
+        // outstanding is below this ratio.
+        value: toWei("0"),
+        isValid: x => {
+          return toBN(x).gte(toBN("0"));
+        }
+      },
```

Comment:
Long term we might want to change this so that this value is specified in terms of percentage of the token price since then this value could be specified such that no valid liquidation would ever lose money.

Reply:
I'll add a todo